slacktee-0.1/debian/slacktee/etc/slacktee.conf

```diff
@@ -0,0 +1,13 @@
+# ----------
+# Configuration
+#   Describes the Incoming Webhook allowing you to post messages into Slack.
+#   After the configuration, copy this file to /etc or your home directory.
+#   NOTE : Please rename this file to '.slacktee', if you'd like to place this in your home directory.
+# ----------
+webhook_url=" https://hooks.slack.com/services/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"      # Incoming Webhooks integration URL. See https://my.slack.com/services/new/incoming-webhook
```

Comment:
You might not want to include your webhool in the package

Reply:
Removed it in my second commit